analyzers/tests/SonarAnalyzer.UnitTest/Rules/Utilities/LogAnalyzerTest.cs

```diff
@@ -56,6 +56,30 @@ public void LogCompilationMessages_CS_NonConcurrent()
         public void LogAutogenerated_VB() =>
             Verify(new[] { "Normal.vb", "GeneratedByName.generated.vb", "GeneratedByContent.vb" }, VerifyGenerated);
 
+        [TestMethod]
+        public void LogAutogenerated_CsHtml()
+        {
+            const string fileName = "Generated_cshtml.g.cs";
+            using var scope = new EnvironmentVariableScope(false) { EnableRazorAnalysis = true };
+
+            CreateBuilder(fileName)
+                .WithSonarProjectConfigPath(AnalysisScaffolding.CreateSonarProjectConfig(TestContext, ProjectType.Product))
+                .VerifyUtilityAnalyzer<LogInfo>(logs => logs.Should().NotContain(x => x.Text.IndexOf(fileName, StringComparison.OrdinalIgnoreCase) != -1));
```

Comment:
Educational: I'm not sure I understand what is going on here:
`logs.Should().NotContain(x => x.Text.IndexOf(fileName, StringComparison.OrdinalIgnoreCase) != -1))`
what are we checking exactly?

Reply:
I've modified all the utility analyzers to not compute any metrics for the .cshtml files since these files are owned by the HTML plugin. Due to this, the "_cshtml.g.cs" will be completely ignored.


The test is now checking that there are no LogInfo messages created that contain the name of the file.

The LogInfo classes are created by https://github.com/SonarSource/sonar-dotnet/blob/master/analyzers/src/SonarAnalyzer.Common/Rules/Utilities/LogAnalyzerBase.cs#L48